spec/mocha.opts

```diff
@@ -0,0 +1,2 @@
+--require spec/setup.js
+--recursive
```

Comment:
I'd rather use a file to encapsulate what is happening than building up a big 1 liner in the package.json. Mocha has [a LOT of options](http://mochajs.org/#usage) so it can get big but I could go either way.